src/components/pages/pseudofill.tsx

```diff
@@ -1,25 +1,98 @@
-import React from 'react';
+import { useState } from 'react';
+import '../../styles/pseudofill.scss';
+import { GeneralDropdown } from '../shared/generalDropdown';
 
 // interface PseudoFillProps {
-//   onCorrect: () => void;
+// onCorrect: () => void;
 // }
 
 function PseudoFill(): JSX.Element {
 
+  const [fillValue1, setFillValue1] = useState('');
```

Comment:
Can we think of a more efficient way to store the values? Perhaps 2 arrays or a 2D array. This will make our code less repetitive and also not limit the number of instructions we have.